x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/action_log_expanded_tray.tsx

```diff
@@ -245,7 +249,7 @@ export const ActionsLogExpandedTray = memo<{
         description: (
           // codeblock for output
           <StyledEuiCodeBlock data-test-subj={getTestId('details-tray-output')}>
-            <OutputContent action={action} data-test-subj={dataTestSubj} />
+            <OutputContent action={{ ...action, command }} data-test-subj={dataTestSubj} />
```

Comment:
This is the actual fix here. Rest of the change is line changes due to updates to `OutputContent` param types